G1ANT.Addon.Net/Commands/MailImapCommand.cs

```diff
@@ -68,14 +67,12 @@ public void Execute(Arguments arguments)
             {
                 ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
             }
-            NetworkCredential credentials = null;
-            if (!string.IsNullOrEmpty(arguments.Login?.Value) && !string.IsNullOrEmpty(arguments.Password?.Value))
-                credentials = new NetworkCredential(arguments.Login.Value, arguments.Password.Value);
+            var authenticator = new SimpleAuthenticationModel(arguments.Login?.Value, arguments.Password?.Value);
```

Comment:
In my opinion we should check all neccessary arguments and give to the RPA Developer information if there is something missing. The same in url.